ui/Cargo.toml

```diff
@@ -39,6 +39,7 @@ opener = "0.5.0"
 soloud = "1.0.2"
 open = "3.2.0"
 wry = { version = "0.23.4" }
+lazy_static = "1.4.0"
```

Comment:
Its usually best to use once_cell over lazy_static since `Lazy` doesnt have to be in a macro and is const.